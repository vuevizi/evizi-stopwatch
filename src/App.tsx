import React from 'react';
import logo from './logo.svg';
import './App.css';
import {MainLayout} from "./layouts";
function App() {
  return (
    <div className="App">
        <MainLayout/>
    </div>
  );
}

export default App;
