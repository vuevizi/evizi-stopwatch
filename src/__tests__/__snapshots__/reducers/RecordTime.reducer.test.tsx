import configureStore from 'redux-mock-store';
import {Provider} from 'react-redux';
import {shallow} from 'enzyme';
import App from '../../../App';
import toJson from 'enzyme-to-json';
const mockStore = configureStore([]);

describe('My Connected React-Redux Component', () => {
  let store;
  let component: any;
  beforeEach(() => {
    store = mockStore({
      amountOfRecords: 0,
      timeRecords: [],
    });
    store.dispatch = jest.fn();
    component = shallow(
      <Provider store={store}>
        <App />
      </Provider>
    );
  });

  it('should dispatch an action on button click', () => {});
  it('should render with given state from Redux store', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
