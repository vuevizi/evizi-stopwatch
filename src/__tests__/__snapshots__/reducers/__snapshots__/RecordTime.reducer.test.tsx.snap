// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My Connected React-Redux Component should render with given state from Redux store 1`] = `
<ContextProvider
  value={
    Object {
      "getServerState": undefined,
      "store": Object {
        "clearActions": [Function],
        "dispatch": [MockFunction],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
      "subscription": Object {
        "addNestedSub": [Function],
        "getListeners": [Function],
        "handleChangeWrapper": [Function],
        "isSubscribed": [Function],
        "notifyNestedSubs": [Function],
        "trySubscribe": [Function],
        "tryUnsubscribe": [Function],
      },
    }
  }
>
  <App />
</ContextProvider>
`;
